pkg/ibmvpc/analysis_output_test.go

```diff
@@ -675,16 +676,16 @@ func (tt *vpcGeneralTest) runTest(t *testing.T) {
 
 	// generate actual output for all use cases specified for this test
 	for _, uc := range tt.useCases {
-		err := runTestPerUseCase(t, tt, vpcConfigs, vpcConfigs2nd, uc, tt.mode, analysisOut, explanationArgs)
+		err := runTestPerUseCase(t, tt, vpcConfigs, uc, tt.mode, analysisOut, explanationArgs)
```

Comment:
`runTestPerUseCase` did not change - I suspect there is a bug that in the current version only the 1st configs is being parsed from main, and that the tests failed to detect the problem since the flow here is different than the one in main. Better have a similar flow so that bugs in the main flow will be detected

Reply:
Im not sure I understand the the comment.
I dont think that there is a bug here, I didnt change the functionality of the code, and diff do work from main.
anyway. this PR is only about editing the multipleVPCConfigs struct.
if you think that there is a problem with the flow of the parser/ main/testing. please address it to a new issue.

Reply:
https://github.com/np-guard/vpc-network-config-analyzer/issues/577 
@adisos fyi